html/YANPIWS.js

```diff
@@ -34,14 +34,21 @@ function loadXMLDoc(URL, targetId, callback) {
 // todo - use class for defaultSize, remove from signature
 function setClockSize(state, defaultSize){
     if (state == 'big'){
-        // todo - put all this in a class and then just add and remove class
-        $('#time').css('font-size', '127pt').css('text-align', 'center').css('width', '100%');
-        $('#date').css('text-align', 'center').css('width', '100%');
+        $('#time').removeClass("small_time")
+        $('#date').removeClass("small_time")    
+        $('#datetimewind').removeClass("small_time_parent")      
+        $('#time').addClass("big_time")
+        $('#date').addClass("big_time")
```

Comment:
Thanks so much for cleaning up my `// todo` !! ;)

Since we're in there refactoring the DOM, I think it'd be cleaner to have just two calls, one to `removeClass()` and one to `addClass()`.  We can achieve this by adding a new class (maybe `timeSizable`?!) and then juts call `$('.timeSizable').addClass("big_time")`.  net result is 5 calls -> 3 calls (rereading this it feels kinda like a nitpick)